src/components/filters/useGroupFilter.test.js

```diff
@@ -97,39 +99,45 @@ describe('some groups available', () => {
           },
         ]
       `);
-    });
   });
 
   it('can enable no group option', async () => {
-    const { result } = renderHook(() => useGroupFilter(true));
-
-    await waitFor(() => {
-      const [config] = result.current;
-      expect(config.filterValues.children).toMatchInlineSnapshot(`
-        <SearchableGroupFilter
-          initialGroups={Array []}
+    const { result, waitForNextUpdate } = renderHook(() =>
+      useGroupFilter(true)
+    );
+
+    await waitForNextUpdate();
+    const [config] = result.current;
+    expect(config.filterValues).toMatchInlineSnapshot(`
```

Comment:
Could we use `toMatchObject` here as well? All other hook tests appear to use it instead of an inline snapshot.